src/Parser/DeathKnight/Blood/Modules/Items/Skullflowers_Haemostasis.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS';
+import ItemDamageDone from 'Main/ItemDamageDone';
+import ItemHealingDone from 'Main/ItemHealingDone';
+import Wrapper from 'common/Wrapper';
+
+class SkullflowersHaemostasis extends Analyzer {
+
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasShoulder(ITEMS.SKULLFLOWERS_HAEMOSTASIS.id);
+  }
+
+  buffStack = 0;
+  wastedBuff = 0;
+  maxBuffStacks = 5;
+  damage=0;
+  heal=0;
+  percentBuff=1/6;
```

Comment:
Shouldn't this be 1/5 (20%)? Also, I'd suggest you make it a global const (define it before the class and name it in all caps). maxBuffStacks could also be made a global const.

Reply:
If i am doing this wrong then sorry. But my understanding of that is as follows. Yes i increases the damage and healing by 20%. But when i see the damage, the buff has already been applied. So i am working backwards to find out from that damage how much of that damage came from the buff.

ex: 100 damage with 20% (20 extra damage then) buff = 120 total damage. 
So working backwards 120 *(1/6) = 20 which gives me the portion that was the damage increased part that i want to track.